pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -413,13 +419,130 @@ func newMigrationMonitor(vmi *v1.VirtualMachineInstance, l *LibvirtDomainManager
 		options:                  options,
 		migrationErr:             migrationErr,
 		progressWatermark:        int64(0),
+		remainingData:            int64(0),
 		progressTimeout:          options.ProgressTimeout,
 		acceptableCompletionTime: options.CompletionTimeoutPerGiB * getVMIMigrationDataSize(vmi),
 	}
 
 	return monitor
 }
 
+func (m *migrationMonitor) shouldTriggerTimeout(elapsed int64, domSpec *api.DomainSpec) bool {
+	if m.acceptableCompletionTime == 0 {
+		return false
+	}
+
+	if domSpec.Metadata.KubeVirt.Migration != nil && domSpec.Metadata.KubeVirt.Migration.Mode == v1.MigrationPostCopy {
+		return false
+	}
+
+	if elapsed > m.acceptableCompletionTime {
+		return true
+	}
+
+	return false
+}
+
+func (m *migrationMonitor) shouldTriggerPostCopy(elapsed int64, domSpec *api.DomainSpec) bool {
+	if m.shouldTriggerTimeout(elapsed, domSpec) && m.options.AllowPostCopy {
+
+		return true
+	}
+	return false
```

Comment:
How about: `return m.shouldTriggerTimeout(elapsed, domSpec) && m.options.AllowPostCopy`

Reply:
personally i like seeing true and false. i don't care either way what someone else does though. unless we adopt some code style guidelines i'm going to leave it.